lib/beaker/options/parser.rb

```diff
@@ -217,6 +217,12 @@ def parse_args(args = ARGV)
         cmd_line_options[:command_line] = ([$0] + args).join(' ')
         @attribution = @attribution.merge(tag_sources(cmd_line_options, "flag"))
 
+        if File.exists?('.beaker.yml')
```

Comment:
It seems like there needs to be an addition to the priority list in the yarddoc for this `parse_args` method. The interesting issue here is that the precedence listed here would indicate project => global => subcommand, subcommand winning out. It isn't linear, per se, but, it actually seems to be more usable to me. Was that intentional to do it that way?

I also think that you can eliminate this `if` block of code and just add `"project" => ".beaker.yml"` to the middle of the options_files just below this. That code handles the file detection and merging.

Reply:
> It seems like there needs to be an addition to the priority list in the yarddoc for this parse_args method.

Ah yup, I'll update that.

The rest makes sense, let me update that.